### PR TITLE
Issue 64 - Schema.TagField is not sortable

### PR DESCRIPTION
**Pull request by rnbWarden**

### CODE REVIEW

File: src/main/java/io/redisearch/Schema.java [hunk 1]
```diff
@@ -105,11 +105,19 @@ public void serializeRedisArgs(List<String> args) {
         private final String separator;
 
         public TagField(String name) {
-        	this(name, DEFAULT_SEPARATOR);
+        	this(name, DEFAULT_SEPARATOR, false);
```
Comment (gkorland) [inline src/main/java/io/redisearch/Schema.java]: why did you need this change?

Comment (rnbWarden) [reply src/main/java/io/redisearch/Schema.java]: For sure it's not strictly required. I felt like chaining the constructors was a bit ugly and going directly to the maximum parameterized constructor was best. I'm not strongly attached to this change tho. Just my personal preference.

Comment (gkorland) [reply src/main/java/io/redisearch/Schema.java]: I prefer it the other way so if we change the default it will effect both.

Comment (rnbWarden) [reply src/main/java/io/redisearch/Schema.java]: sure... makes sense
